src/script/view_model/VideoCallingViewModel.js

```diff
@@ -195,15 +195,18 @@ z.viewModel.VideoCallingViewModel = class VideoCallingViewModel {
     this.TimeUtil = TimeUtil;
   }
 
-  chooseSharedScreen(conversationId) {
+  async chooseSharedScreen(conversationId) {
```

Comment:
Not needed and not supported
```suggestion
  chooseSharedScreen(conversationId) {
```